MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -2077,11 +2086,15 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 	// Mark the end of the archive
 	cacheEntryType = MVKPipelineCacheEntryTypeEOF;
 	writer(cacheEntryType);
+#else
+	MVKAssert(false, "Pipeline cache serialization is unavailable. To enable pipeline cache serialization, build MoltenVK without the MVK_EXCLUDE_CEREAL build setting.");
```

Comment:
`MVKAssert` is okay here, because it would be a MoltenVK code error to call this function when CEREAL is not available.

Reply:
Ok.